packages/loader/driver-utils/src/blobAggregationStorage.ts

```diff
@@ -188,15 +182,20 @@ export class BlobAggregationStorage extends SnapshotExtractor implements IDocume
 
     protected virtualBlobs = new Map<string, ArrayBufferLike>();
 
-    static wrap(storage: IDocumentStorageService, logger: ITelemetryLogger, allowPacking = gatesAllowPacking()) {
+    static wrap(
+        storage: IDocumentStorageService,
+        logger: ITelemetryLogger,
+        allowPacking = gatesAllowPacking(),
+        packingLevel = 2,
```

Comment:
It feels like it needs to be required argument, not optional

Reply:
it's optional during one use by fetch tool...